src/specs/components/gallery/demo/data.js

```diff
@@ -167,10 +170,10 @@ module.exports = {
       title: 'About the EU',
       description: 'About the EU',
       meta: 'Copyright, Author, Licence for image 9',
-      share_path: '/share#example-imag9e.jpg',
+      share_path: '/share#example-image9.jpg',
     },
     {
-      // Image
+      // Image without thumbnail
```

Comment:
maybe it would be easier for the users if we apply the thumbnail to the "majority" of the items we demo, otherwise they will have to look for the one that has a different image defined as the thumbnail and the one in the picture tag. I mean, i would keep like one without, just to test the backward compatibility of what we are doing.

Reply:
I thought about it, but we don't have any more images available, and reusing them could be confusing.
I will put the thumbnail on the first image, so that's more visible